tests/zfs-tests/tests/functional/cli_root/zfs_get/zfs_get_list_d.kshlib

```diff
@@ -65,7 +65,6 @@ function depth_fs_setup
 					log_must zfs create -V 8M $fs/vol_"$j"_depth"$i"
 				fi
 				log_must zfs snapshot $fs@snap_"$j"_depth"$i"
-				log_must zfs bookmark $fs@snap_"$j"_depth"$i" '#bookmark'_"$j"_depth"$i"
```

Comment:
Why is this change reverted?

Reply:
dont know. never touched this file. not my patch